package.json

```
@@ -68,7 +68,7 @@
   },
   "dependencies": {
     "@apidevtools/json-schema-ref-parser": "^9.0.6",
-    "@asyncapi/specs": "^2.7.7",
+    "@asyncapi/specs": "^2.7.8",
```

Since we don't know which version will be the one containing the changes in the `@asyncapi/specs` package, let's point directly to the branch of the other PR:

```suggestion
    "@asyncapi/specs": "git://github.com/asyncapi/asyncapi-node.git#security-scheme-for-kafka",
```